src/use-memo/use-memo.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller } from 'stimulus'
+
+const memoize = (controller: Controller, name: string, value: any) => {
+  Object.defineProperty(controller, name, { value })
+  return value
+}
+
+export const useMemo = (controller: Controller) => {
+  ;(controller.constructor as any).memos?.forEach((getter: string) => {
+    memoize(controller, getter, Reflect.get(controller, getter))
```

Comment:
ok looks like Reflect is not supported on IE11
My initial plan was to try to get the same level of browser compatibility as Stimulus. I got a free open source saucelab account that I still need to finish setting up
So either we add a polyfill to the documentation 
https://github.com/tvcutsem/harmony-reflect (and to our test environment)
or 
do something like that `(controller as any)[getter]`

solution B looks easier